### PR TITLE
make logging configurable via helm

### PR DESCRIPTION
**Pull request by kishansairam9**

### CODE REVIEW

File: helm/templates/log-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+data:
+  log4j.properties: |-
+    log4j.rootLogger={{ .Values.logging.level }}, stdout
+
+    log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+    log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+    log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
+
+    log4j.logger.kafka=ERROR, stdout
+    log4j.additivity.kafka.server=false
```
Comment (laxmanchekka) [inline helm/templates/log-configmap.yaml:16]: @kishansairam9 : fixed duplication issues with additivity in pinot log configuration
can you please refer to that and fix this as well and rebase the PR with main branch.

Please note that pinot is using log4j2 while schema registry is using log4j-1.2

Comment (kishansairam9) [reply helm/templates/log-configmap.yaml:16]: done

File: helm/templates/log-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
```
Comment (laxmanchekka) [inline helm/templates/log-configmap.yaml:16]:
```suggestion
    log4j.additivity.kafka=false
```

File: helm/templates/log-configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  name: {{ template "schema-registry.fullname" . }}-log-configmap
+  labels:
+    {{- include "schema-registry.labels.standard" . | nindent 4 }}
+data:
+  log4j.properties: |-
+    log4j.rootLogger={{ .Values.logging.level }}, stdout
+
+    log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+    log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+    log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
```
Comment (laxmanchekka) [inline helm/templates/log-configmap.yaml:13]: Logging class name and line number is costly.
Make this equivalent with the following which is tested and used in pinot. However, please note that the following pattern is from log4j2.

`%d{yyyy-MM-dd HH:mm:ss.SSS} [%level] [%thread] [%c{1.}] - %msg%n`

File: helm/templates/log-configmap.yaml (hunk 1 above)
Comment (kishansairam9) [reply helm/templates/log-configmap.yaml]: I checked the syntax for log4j 1 and wrote this. We can't test this until we merge

File: helm/templates/log-configmap.yaml (hunk 3 above)
Comment (kishansairam9) [reply helm/templates/log-configmap.yaml:13]: Removing %L should be good then? We anyway seem to log class name

Comment (laxmanchekka) [inline helm/templates/log-configmap.yaml:13]:
```suggestion
    log4j.appender.stdout.layout.ConversionPattern=[%d] [%p] [%t] [%c] - %m%n
```

**Review by laxmanchekka — APPROVED** [review]

**Pull request merged**